src/status_im2/contexts/profile/data/events.cljs

```diff
@@ -0,0 +1,157 @@
+(ns status-im2.contexts.profile.data.events
+  (:require [clojure.string :as string]
+            [native-module.core :as native-module]
+            [quo2.foundations.colors :as colors]
+            [re-frame.core :as re-frame]
+            [react-native.platform :as platform]
+            [status-im.multiaccounts.update.core :as multiaccounts.update]
```

Comment:
why do we still keep multiaccounts? or we'll have more PRs on multiaccounts? and we move only core in this PR